Gemfile

```diff
@@ -4,6 +4,8 @@ source 'https://rubygems.org'
 
 gem 'cocoapods', '~> 1.11'
 gem 'cocoapods-check', '~> 1.1'
+gem 'danger', '~> 8.6'
+gem 'danger-rubocop', '~> 0'
```

Comment:
The latest `danger-rubocpo` version is `0.10.0`. Using `~> 0` will resolve to the latest version `0.x` compatible with the repo.

Here's an example of the behavior in action using `~> 0` for `danger`:

![image](https://user-images.githubusercontent.com/1218433/213323412-ab1e1ccd-15f2-4abc-aafe-8ec97cbb831d.png)